### ISSUE TITLE
Chart is not responsive

### ISSUE THREAD

**Issue by atgillette** [highcharts/highcharts-angular#166]
> For general tech support, please see www.highcharts.com/support.
> Please report only issues about `highcharts-angular` wrapper or content of this repository.
> For general issues with Highcharts and TypeScript the correct repository to report such issues is [main Highcharts repository](https://github.com/highcharts/highcharts/issues).

### Describe the bug
The highchart angular does not resize the width when the resizing the browser.

### Expected behavior
The highchart angular resizes the width when the resizing the browser.

### Demo
https://codesandbox.io/s/angular-8-highcharts-xekz0

#### Steps required to recreate the problem in the demo:
1. ...

### Setup used
 - NodeJS version ...
 - Angular version ... 8.1.1
 - TypeScript version ...
etc.

**Comment by atgillette** [comment]
I found the cause of the issue.  It turns out that html table element was not properly resizing.  To remedy the problem, I just simply need to use "table-layout: fixed;" style on the table element.

**Comment by luigi198** [comment]
Hi,

Right now I'm having this issue, the chart is not responsive for me. I followed the demo setting width: 100% to the style of the chart:

`<highcharts-chart
  [Highcharts]="Highcharts"
  [options]="chartOptions"
  [(update)]="updateChart"
  style="width: 100%;"
>
</highcharts-chart>`

The HTML parent element is a div with float: left, width: 33%. I can confirm the div resizes but the graph don't. I had to implement a work around that sets an Observable listening to the resize window event and update the graph with the corresponding width of the parent.

Am I missing something?
